### ISSUE TITLE
Bigtable retention

### ISSUE THREAD

**Issue by hexedpackets** [spotify/heroic#445]
Metrics stored in Bigtable should be able to have a retention policy set via a column family garbage collection policy.

**Comment by hexedpackets** [comment]
One major edge case to consider is if the node is configured with a different policy than the rest of the cluster. How do we decide when to update the Bigtable cluster? One option is to only set a new policy, never override an existing one. If we go that route we should have a shell task for manually adjusting the policy later.

**Comment by dmichel1** [comment]
Good point - we could have the default policy be forever (current behavior) and have the GC policy always be set through a shell task. It would be good to have some best practices on setting up a new cluster. e.g. these one-off shell commands should be in SCM and run though a CI process that way this is always repeatable and we have a source of truth for new clusters
